Sources/Datadog/Core/System/MobileDevice.swift

```diff
@@ -99,3 +100,29 @@ internal class MobileDevice {
         }
     }
 }
+
+private class LowPowerModeMonitor {
```

Comment:
Hey guys 👋

Quick question, does this class needs to be open to subclassing? Why not putting it as `final`?

Reply:
I'm fairly sure this is implicit for a `private` class, however always best to be certain so I've added `final` too 👍